Civi/Funding/PayoutProcess/Validator/DrawdownValidator.php

```diff
@@ -63,6 +67,27 @@ public function __construct(FundingCaseManager $fundingCaseManager, PayoutProces
    * phpcs:disable Drupal.Commenting.FunctionComment.IncorrectTypeHint
    */
   public function validate(AbstractEntity $new, AbstractEntity $current): EntityValidationResult {
+    $payoutProcess = $this->getPayoutProcess($new);
+    $this->assertNotClosed($payoutProcess);
+
+    if ($new->getAmount() < 0) {
+      $payoutProcess = $this->getPayoutProcess($new);
+      $fundingCase = $this->getFundingCase($payoutProcess);
+
+      if (!$this->hasReviewPermission($fundingCase->getPermissions())) {
+        return $this->createAmountLessThanZeroResult();
```

Comment:
handle this case as on line 109 and 113?

Reply:
Contacts with `drawdown_create` permission cannot edit drawdowns, i.e. they aren't able to request an update request at all, so we don't need to check that permission here.